src/test/java/net/oauth/jsontoken/JsonTokenParserTest.java

```diff
@@ -227,4 +199,47 @@ public void testPublicKey() throws Exception {
       // expected
     }
   }
+
+  private boolean testVerify_timeFrame(Instant issuedAt, Instant expiration) throws Exception {
```

Comment:
Prefer to use the "test" prefix and underscore method naming for the actual test methods, not for the helper methods.

Reply:
Renamed